### PR TITLE
SAA-32 updating build and configuration to kick off DB migrations on service start.

### PR DESCRIPTION
**Pull request by matt-crowson-moj**

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -18,6 +18,28 @@ spring:
...
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+    show-sql: false
+    generate-ddl: false
+    hibernate:
+      ddl-auto: none
+
+  datasource:
+    url: 'jdbc:postgresql://${DB_SERVER}/${DB_NAME}?sslmode=prefer'
```
Comment (matt-crowson-moj) [inline src/main/resources/application.yml:32]: So this is set to prefer for SSL at the mo but we may want to change it (make it configurable??)

File: src/main/resources/banner.txt [hunk 2]
```diff
@@ -1,13 +1,8 @@
...
-___ ____ _  _ ___  _    ____ ___ ____ 
- |  |___ |\/| |__] |    |__|  |  |___ 
- |  |___ |  | |    |___ |  |  |  |___ 
-                                      
-_  _ ____ ___ _    _ _  _ 
-|_/  |  |  |  |    | |\ | 
-| \_ |__|  |  |___ | | \| 
-                          
-Please change me by generating your own ASCII art and placing in banner.txt
+                    _     _           _   _     _                                                                                                _                  _____    _____
```
Comment (matt-crowson-moj) [inline src/main/resources/banner.txt:1]: Obvs nothing to do with DB changes but thought may as well get it updated.

File: src/main/resources/application.yml (hunk 1 above)
Comment (timharrison-moj) [reply src/main/resources/application.yml:32]: Yes, ok for now. Usually set to `sslmode=verify-full`

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**